application/console/migrations/m190411_201535_add_nag_dates_to_user_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m190411_201535_add_nag_dates_to_user_table
+ */
+class m190411_201535_add_nag_dates_to_user_table extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->addColumn('{{user}}', 'nag_for_mfa_after', 'date not null');
```

Comment:
I like that you included `after` here, very helpful!

Reply:
Thanks.